### ISSUE TITLE
fix output for commands that use backspace for output

### ISSUE THREAD

**Issue by deanmalmgren** [deanmalmgren/flo#53]
commands like `curl` display progress by "backspacing" their output. [This isn't handled terribly well by our teeing mechanism at the moment](https://github.com/deanmalmgren/flo/blob/master/flo/shell.py#L14-L16), and it makes it difficult to see how long a `curl` command might take.

**Comment by deanmalmgren** [comment]
subprocess annoyingly writes output using 

``` python
input_offset += os.write(fd, chunk)
```

instead of just calling the write method on the file object that is passed into the PIPE. wtf.

To make matters worse, it is [really damn clunky to suppress newlines](http://stackoverflow.com/a/13373385/564709) when working with the logging module.

This made it so that the only reasonable way of handling backspacing outputs is to have a custom logger and to read output from the subprocess pipe in byte size chunks (literally).

In short, no more logging module.
